pages/shared/[folderId]/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import Banner from "../../../components/domains/shared/Banner";
+import CardList from "../../../components/commons/CardList";
+import SearchInput from "../../../components/commons/SearchInput";
+import styles from "../../../styles/sharedPage.module.css";
+import { Link } from "../../../types/common";
```

Comment:
경로에 따라 길어지는 path를 TS컴파일러옵션(tsconfig.json)을 수정해서 개선해보시면 좋겠습니다.

```suggestion
import { Link } from "@/types/common";
```

https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases